app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -101,10 +103,16 @@ export const FixedSessionsMapCtrl = (
     angular.element(document).ready(function () {
       const node = document.getElementById('newFixedFilters');
 
+      const timeRange = {
+        timeFrom: $scope.params.get('data').timeFrom,
+        timeTo: $scope.params.get('data').timeTo,
+      };
+
       const flags = {
         tags: $scope.params.get('data').tags.split(', ').filter((tag) => tag !== "") || [],
-        profiles: $scope.params.get('data').usernames.split(', ').filter((tag) => tag !== "") || []
-      }
+        profiles: $scope.params.get('data').usernames.split(', ').filter((tag) => tag !== "") || [],
+        timeRange: timeRange
```

Comment:
When key and value have the same name you can just have `timeRange` (instead of `timeRange: timeRange`)